client/blocks/product-purchase-features-list/find-new-theme.jsx

```diff
@@ -17,9 +17,9 @@ export default localize( ( { selectedSite, translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon={ <img alt="" src="/calypso/images/upgrades/customize-theme.svg" /> }
-				title={ translate( 'Try a New Theme' ) }
+				title={ translate( 'Premium themes' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **43** times:
`translate( 'Premium Themes' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).